src/system/Button/Button.js

```diff
@@ -34,8 +34,7 @@ const Button = React.forwardRef( ( { disabled, onClick, sx, ...props }, forwardR
 				'&:hover': {
 					textDecoration: 'none',
 				},
-				'&:focus': theme => theme.outline,
-				'&:focus-visible': theme => theme.outline,
+				'&focus-visible': theme => theme.outline,
```

Comment:
Did you miss the `:`?

Reply:
Yeah you're right

Reply:
Added in 3dbe67165889b810251386f47f59d48349b9950d